debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -386,8 +400,13 @@ protected Object convertJson(Column column, Field fieldDefn, Object data) {
                         r.deliver(JsonBinary.parseAsString((byte[]) data));
                     }
                     catch (IOException e) {
-                        parsingErrorHandler.error("Failed to parse and read a JSON value on '" + column + "' value " + Arrays.toString((byte[]) data), e);
-                        r.deliver(column.isOptional() ? null : "{}");
+                        if (eventConvertingFailureHandlingMode == FAIL) {
```

Comment:
@jpechane 
I removed `parsingErrorHandler` because it looks same with `eventConvertingFailureHandlingMode`.
but to use the mode, MysqlValueConverters needs the value of mode in constructor.